lib/stack_master/parameter_resolvers/latest_ami.rb

```diff
@@ -0,0 +1,18 @@
+module StackMaster
+  module ParameterResolvers
+    class LatestAmi < Resolver
+      array_resolver class_name: 'LatestAmi'
```

Comment:
This should be a different name to what the singular class name is. Array resolvers work like this:

```
latest_amis:
  - tag:application=blah
  - tag:name=blah
```

and the resulting parameter should be a comma separated list of AMI's `ami-123,ami-456`.